runtime/themes/catppuccin.toml

```diff
@@ -91,3 +91,8 @@ gray_2 = "#C3BAC6"
 white = "#D9E0EE"
 lavender = "#C9CBFF"
 rosewater = "#F5E0DC"
+
+silver = "#c0c0c0"
+apricot = "#f47868"
+lightning = "#ffcd1c"
```

Comment:
These three keys are from the default theme file, probably a remnant from being copied. The catppuccin [style guide](https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#typography) recommends using `maroon` for errors and `peach` for warnings (which is where these palette keys are used in this file). Something mild like `lavender` would be a good fit for the hint key too.